db-migration/target/classes/db/migration/V1__table_customers_create_script.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS customers(
```

Comment:
Why sql scripts are added to db-migration/target ?

Reply:
Added db-migration/target to gitignore